retropie_packages.sh

```diff
@@ -25,11 +25,13 @@ if [[ -n "$__user" ]]; then
     fi
 else
     user="$SUDO_USER"
+    __user="$SUDO_USER"
     [[ -z "$user" ]] && user="$(id -un)"
-    [[ -z "$group" ]] && group="$(id -gn)"
+    [[ -z "$__user" ]] && __user="$(id -un $SUDO_USER)"
```

Comment:
Please can you detail the need for the addition of the `$SUDO_USER` env variable here ?